nilearn/reporting/_plot_matrices.py

```diff
@@ -62,6 +62,87 @@ def plot_design_matrix(design_matrix, rescale=True, ax=None, output_file=None):
     return ax
 
 
+def plot_event(model_event, cmap=None, output_file=None, **fig_kwargs):
+    """Creates plot for event visualization.
+
+    Parameters
+    ----------
+    model_event : DataFrame or list of DataFrame
+        the list of Dataframe obtained by ```first_level_models_from_bids```
+        or the Dataframe obtained by reading a BIDS *_event.tsv
+
+    cmap : str or matplotlib.cmap, optional
+        the colormap used to label different events
+
+    output_file: string or None, optional,
+        The name of an image file to export the plot to. Valid extensions
+        are .png, .pdf, .svg. If output_file is not None, the plot
+        is saved to a file, and the display is closed.
+
+    **fig_kwargs : extra keyword arguments, optional
+        Extra arguments passed to matplotlib.pyplot.subplots
+
+    Returns
+    -------
+    Plot Figure object
+
+    """
+    import matplotlib.patches as mpatches
+    import pandas as pd
```

Comment:
Is there a reason why those imports are here, rather than at the module level?

Reply:
I forgot to remove it! I will add it at module level!